packages/cloud-functions/src/cico/composeProviderUrls.ts

```diff
@@ -0,0 +1,83 @@
+import crypto from 'crypto'
+import {
+  CASH_IN_SUCCESS_DEEPLINK,
+  CASH_IN_SUCCESS_URL,
+  MOONPAY_DATA,
+  RAMP_DATA,
+  TRANSAK_DATA,
+  VALORA_LOGO_URL,
+  XANPOOL_DATA,
+} from '../config'
+import { ProviderRequestData, Providers } from './fetchProviders'
+const URL = require('url').URL
+
+export const composeProviderUrl = (provider: Providers, requestData: ProviderRequestData) => {
```

Comment:
super nit: This const is called `composeProviderUrl` but the file name is `composeProviderUrls` in plural. Any of the two is fine, but it should probably be the same?

Reply:
Good catch. Changed it.